src/components/ContributionAssistantPriceFormCard.vue

```diff
@@ -2,12 +2,20 @@
 <template>
   <v-card
     class="mb-4"
-    title="Label"
-    prepend-icon="mdi-tag-outline"
     height="100%"
     style="border: 1px solid transparent"
-    :color="productPriceForm.processed ? 'success' : ''"
   >
+    <v-card-item>
+      <template #prepend>
```

Comment:
looking at a similar UX in `AddPriceMultiple.ProductPriceDetails.NewPrice` i think we can simplify this bit, and keep the label & prepend-icon in the `<v-card>` definition, no ?

Reply:
Definitely ! Thanks for the reference.

For some reason, when I tried that yesterday, the delete icon wouldn't align correctly. Could be because I was trying to use a button instead of a clickable v-icon.